internal/reader/ccip.go

```diff
@@ -239,27 +239,17 @@ func (r *CCIPChainReader) MsgsBetweenSeqNums(
 		return nil, err
 	}
 
+	type SendRequestedEvent struct {
```

Comment:
Lets add a TODO / create a ticket in the common epic to look into using modifiers instead of this.

Reply:
the easiest way would be to modify the ABI string to so that it matches just cciptypes.message

Reply:
cciptypes.Message is the "Any2Any" message type for CCIP so we'll likely always have to do some translation from whats onchain to whats in cciptypes.Message, but we can figure that our later